actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/TextbookTocActor.java

```diff
@@ -674,8 +681,8 @@ private void throwInvalidDialCodeError(Set<String> invalidDialCodes) {
                 });
           }
           Map<String, Object> map = new HashMap<>();
-          if (JsonKey.UPDATE.equalsIgnoreCase(mode) && StringUtils.isNotBlank(record.get(id))) {
-            String identifier = record.get(id).trim();
+          if (JsonKey.UPDATE.equalsIgnoreCase(mode) && StringUtils.isNotBlank(trimMapping.get(id))) {
+            String identifier = trimMapping.get(id).trim();
```

Comment:
no need of duplicate trimming, all the values in trimMapping is already trimmed.